buildmaster/old_new_porting_map.yml

```diff
@@ -0,0 +1,388 @@
+# Mapping between old and new commondata for oldcommondata_porter
+#
+# <old_commondata_name>:
+#   new_name: <new_name>
+#   reference_arxiv: <http://etc>
+#   reference_hepdata: <http://etc>
+#
+
+# HERA
+HERACOMBCCEM:
+  new_name: HERA_CC_318GEV_EM-SIGMARED
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t7
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMBCCEP:
+  new_name: HERA_CC_318GEV_EP-SIGMARED
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t6
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMBNCEM:
+  new_name: HERA_NC_318GEV_EM-SIGMARED
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t5
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMBNCEP460:
+  new_name: HERA_NC_225GEV_EP-SIGMARED
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t4
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMBNCEP575:
+  new_name: HERA_NC_251GEV_EP-SIGMARED
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t3
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMBNCEP820:
+  new_name: HERA_NC_300GEV_EP-SIGMARED
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t2
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMBNCEP920:
+  new_name: HERA_NC_318GEV_EP-SIGMARED
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t1
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMB_SIGMARED_B:
+  new_name: HERA_NC_318GEV_EAVG_BOTTOM-SIGMARED
```

Comment:
About this I'm not super happy, but I'm not sure if we can or want to do something ... the thing is this is very related to `HERA_NC_318GEV_*` ...
- here the name is however split in a different way, i.e. `EAVG` is not part of the observable name
- the physical events (in the collider) are the same as in the fully inclusive case, but this is a different analysis (and thus of course a different paper)

Reply:
At some point we have to accept that either we truly do one dataset - one folder (like in the old commondata) or there will always be some ambiguity.